visualization/app/codeCharta/util/dataMocks.ts

```diff
@@ -1,17 +1,18 @@
 import {
```

Comment:
After this PR is cleaned up, we should remove the `VALID` prefix from every variable here, since there are no invalid cases given. Even if we will declare some, we can prefix them with INVALID. It's just not needed to do it for valid stuff.

Reply:
I think this should be done as a separate Merge Request, not really part of the feature.
Or am I in the wrong here?

Reply:
Agreed